components/threading/channel_threads/thread_footer/__snapshots__/thread_footer.test.tsx.snap

```diff
@@ -630,7 +630,7 @@ exports[`components/threading/channel_threads/thread_footer should report total
       <injectIntl(Timestamp)
         day="numeric"
         dispatch={[Function]}
-        hour12={true}
+        hourCycle="h12"
```

Comment:
Has anything change recently with Intl/ICU support in Jest envs? I was trying to trace down why `hourCycle` support was missing (why this change in thread_footer.test.tsx.snap keeps flipping)

I think our current Jest env/setup + Node 10 causes missing `DateTimeFormat...hourCycle` support. I'm guessing it runs fine on Node 14.

Reply:
It's Node 14 vs Node 10. I've been running Node 14 locally, so that's why the update changed this. That's partly what's breaking CI right now though.